README.rst

```diff
@@ -67,6 +67,12 @@ error messages:
 
    $ python -c 'import mne_bids'
 
+For full functionality of ``mne-bids``, you will also need to install the
+following packages:
+
+- nibabel
+- nilearn
```

Comment:
Is `nilearn` a dependency? It's just exposed in an example for plotting ...

If you import any part of `mne-bids` and the import fails when `nilearn` is not present, then I would call it a dependency.

Reply:
I am not saying that it's a dependency, the paragraph says:


> For full functionality of ``mne-bids``, you will also need to install the
following packages:
- nibabel
- nilearn
- matplotlib